goodkey/good_key.go

```diff
@@ -237,15 +237,16 @@ func (policy *KeyPolicy) goodKeyRSA(key *rsa.PublicKey) (err error) {
 	if modulusBitLen%8 != 0 {
 		return berrors.MalformedError("key length wasn't a multiple of 8: %d", modulusBitLen)
 	}
-	// The CA SHALL confirm that the value of the public exponent is an
-	// odd number equal to 3 or more. Additionally, the public exponent
-	// SHOULD be in the range between 2^16 + 1 and 2^256-1.
-	// NOTE: rsa.PublicKey cannot represent an exponent part greater than
-	// 2^32 - 1 or 2^64 - 1, because it stores E as an integer. So we
-	// don't need to check the upper bound.
-	if (key.E%2) == 0 || key.E < ((1<<16)+1) {
-		return berrors.MalformedError("key exponent should be odd and >2^16: %d", key.E)
+
+	// Rather than support arbitrary exponents, which significantly increases
```

Comment:
Let's additionally quote the BR text that was here before, and indicate that we satisfy that by only allowing one exponent, which is in the acceptable range.